Sources/AppcuesKit/Presentation/UI/Components/AppcuesText.swift

```diff
@@ -21,5 +21,6 @@ internal struct AppcuesText: View {
             .applyTextStyle(style)
             .setupActions(on: viewModel, for: model)
             .applyAllAppcues(style)
+            .fixedSize(horizontal: false, vertical: true)
```

Comment:
Yep. You can see an example here (note this example has a fixed width and height, otherwise the height would expand to fit the text) https://github.com/appcues/appcues-mobile-experience-spec/pull/98/files#diff-293429e1144f75bbe866cf9647212a6ec9a7ca883e04437e9afb9acd259c3b7b

And another helpful reference: https://stackoverflow.com/a/57677746